### ISSUE TITLE
Consider only output to LOG instead of hardcoded stdout/stderr

### ISSUE THREAD

**Issue by PragmaTwice** [apache/kvrocks#662]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-kvrocks/issues) and found no similar issues.

### Motivation

Refer to https://github.com/apache/incubator-kvrocks/pull/660#issuecomment-1165637383.

It is good to keep kvrocks with only one output location.
But currently, kvrocks also outputs some information to stdout when logdir is set to a regular file, which can be confusing for users.

### Are you willing to submit a PR?

- [ ] I'm willing to submit a PR!

**Comment by tisonkun** [comment]
cc @git-hulk 

<img width="1728" alt="image" src="https://user-images.githubusercontent.com/18818196/197383662-02ed392d-69be-447a-82e6-f2b0c0cd6870.png">

Currently, I found only these three lines in the source code that we print to the stdout in starting up logic. I don't know whether we should have a rule that only output to LOG everywhere and the current status looks good to me.

**Comment by PragmaTwice** [comment]
Actually something is unexpectedly written to stderr:

![image](https://user-images.githubusercontent.com/20042607/197385705-cc5dbcf9-fed2-4535-8cbd-9633c2ceeb01.png)

So it need some investigation.

**Comment by git-hulk** [comment]
> cc @git-hulk
> 
> <img alt="image" width="1728" src="https://user-images.githubusercontent.com/18818196/197383662-02ed392d-69be-447a-82e6-f2b0c0cd6870.png">
> 
> Currently, I found only these three lines in the source code that we print to the stdout in starting up logic. I don't know whether we should have a rule that only output to LOG everywhere and the current status looks good to me.

To be honest, we don't have an explicit rule on when should send logs to the stdout instead of LOG file, but I think all logs should go into LOG except:
- command outputs like `./kvrock --help`
- logs before the logging instance was setup

**Comment by git-hulk** [comment]
> Actually something is unexpectedly written to stderr:
> 
> ![image](https://user-images.githubusercontent.com/20042607/197385705-cc5dbcf9-fed2-4535-8cbd-9633c2ceeb01.png)
> 
> So it need some investigation.

Yes, I searched the keyword `cout` and found some places are still using `std::cout`. 
And it's weird in this case, coz I find `Logging` is always initialized before opening DB.
